spark-tpcds-benchmark-runner/var/conf/config.yml

```diff
@@ -1,50 +1,101 @@
+dataGeneration:
+  gensort:
+    # Whether to generate data for the sort benchmark.
+    enabled: true
+  tpcds:
+    # Whether to generate data for the TPC-DS benchmark.
+    enabled: true
+  tempWorkingDir: /tmp/spark-benchmark/datagen-working-dir
+  
+  # Whether to overwrite any previously generated data. If set to false, the program will fail with an exception.
+  overwriteData: false
+  
+  # The number of threads used for data generation.
+  parallelism: 5
+
+benchmarks:
+  # Optional. Defaults to current timestamp.
+  experimentName: experiment-1
+  
+  gensort:
+    # Whether to run the sort benchmark.
+    enabled: false
+  tpcds:
+    # Whether to run the TPC-DS benchmark suite.
+    enabled: false
+
+  # The number of iterations to run each benchmark.
+  iterations: 10
+
+# A list of sizes in Gigabytes to generate the data and run the benchmarks.
+dataScalesGb:
+  - 10
+  - 1024
+
 hadoop:
-  hadoopConfDirs: []
-  hadoopConf: {}
-  defaultFilesystem: local
+  # A map of a filesystem name to its settings. The names are used to reference the filesystem in the settings below.
+  # Supported types are "azure", "s3a" and "simple". Use "simple" for HDFS and local file systems.
   filesystems:
+    azure:
+      accessKey: foo
+      accountName: bar
+      containerName: spark-benchmark
+      type: azure
     local:
+      baseUri: 'file:///home/palantir/spark-benchmark'
       type: simple
-      baseUri: file:///tmp/spark-benchmark
-    hdfs:
-      type: simple
-      baseUri: hdfs://spark-benchmark
     s3a:
+      baseUri: 's3a://spark-benchmark/'
+      accessKey: foo # Optional
```

Comment:
let's replace e.g., `foo` and `bar` with a more explicit placeholder like `__PLACEHOLDER__`